app/views/shipit/commits/_commit.html.erb

```diff
@@ -17,6 +17,15 @@
       <p class="commit-meta">
         <%= timeago_tag(commit.committed_at, force: true) %>
       </p>
+      <p class="commit-meta">
+        <% if commit.rollbackable%>
+          <span class='rollbackable-yes'> Safe to rollback </span>
```

Comment:
For a boolean input, a checkbox would make much more sense IMO.

Reply:
hmm could you expand on this? I can't quite visualize what you're describing 😛